pkg/redis/client.go

```diff
@@ -533,3 +768,38 @@ func (c *redisClient) execute(ctx context.Context, wrappedCmd func() ErrCmder) (
 		return cmder, cmder.Err()
 	})
 }
+
+func (c *redisClient) toGosolineZs(zs []baseRedis.Z) []Z {
+	result := make([]Z, len(zs))
+	for i := range zs {
+		result[i] = Z{
+			Score:  zs[i].Score,
+			Member: zs[i].Member,
+		}
```

Comment:
Wouldn't this work as well?
```suggestion
		result[i] = Z(zs[i])
```

Reply:
Done